packages/test/snapshots/README.md

```diff
@@ -117,6 +118,10 @@ Whenever the snapshot format changes in the runtime, the existing snapshots in t
   npm run test:update
   ```
   This will save the current snapshots in the `src_snapshots\<current_snapshots_version>` directory. It will then update the current snapshots under the `current_snapshots` directory to the new format.
+- However, there is a chance that not all the snapshots will be updated. You will see this as a failure in the Azure build, which will indicate which files contain snapshots that were not updated. The failures might have to do with the same property/value and is likely present in all the files within that subdirectory. To solve this, do the following:
```

Comment:
Can you give an example for this failure? I am curious what is leading this to happen. Not sure I like the step of manually updating the snapshots as that can lead to a lot of issues later.

Reply:
So my code change was to remove all instances of summaryCount. After running the test:update script, there were still a few instances where summaryCount was expected. But you're right, manually updating the snapshots is definitely not ideal. 
https://dev.azure.com/fluidframework/public/_build/results?buildId=80731&view=ms.vss-test-web.build-test-results-tab&runId=1765448&resultId=100003&paneView=debug